ddclient

```diff
@@ -3827,70 +3827,97 @@ EoEXAMPLE
 ######################################################################
 ## nic_freedns_update
 ##
-## written by John Haney
```

Comment:
I am not sure if we want to keep those or not. Generally I am not a big fan of them cause most sections are copied from other providers.

Reply:
I'm not a fan of them either for that reason and because they become less and less true over time. Credit should go in the ChangeLog and commit messages.